src/environment.cpp

```diff
@@ -37,6 +37,9 @@ Environment::Environment(IGameDef *gamedef):
 	m_cache_active_block_mgmt_interval = g_settings->getFloat("active_block_mgmt_interval");
 	m_cache_abm_interval = g_settings->getFloat("abm_interval");
 	m_cache_nodetimer_interval = g_settings->getFloat("nodetimer_interval");
+
+	m_time_of_day = g_settings->getU32("world_start_time");
```

Comment:
This means time of day is doubly initialised, right?

Reply:
Just checked, it is. You should remove the inline initializer from the header file.

Reply:
Ok.